docs/source/matplotlib/advanced_plotting.rst

```diff
@@ -139,16 +139,18 @@ visualising vector fields:
 :meth:`streamplots <cartopy.mpl.geoaxes.GeoAxes.streamplot>` (:ref:`example <examples-streamplot>`)
 each with their own benefits for displaying certain vector field forms.
 
-.. literalinclude:: /examples/arrows.py
-.. plot:: examples/arrows.py
+.. figure:: ../gallery/vector_data/images/sphx_glr_arrows_001.png
```

Comment:
These are updated so that old figures generated with the `plot` directive are now linking to the sphinx-gallery image (the end result is basically the same)